helm/micro-integrator/templates/logstash-elasticsearch-creds.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{ if .Values.wso2.centralizedLogging.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+    name: logstash-elasticsearch-creds
+    namespace : {{ .Release.Namespace }}
+type: Opaque
+data:
+    username: {{ default "elastic" .Values.wso2.centralizedLogging.logstash.elasticsearch.username | b64enc }}
```

Comment:
@ThilinaManamgoda shall we avoid setting default values in the resource definitions and set default values in the `values.yaml` itself?

Reply:
Please check all applicable cases.

Reply:
Done